nodes/src/nodes/agent_crewai/crewai.py

```
@@ -161,25 +185,73 @@ def _run(self, input: Any = None, **kwargs: Any) -> str:
         return tools
 
+
+# ── CrewDriver ────────────────────────────────────────────────────────────────
+
+
+class CrewDriver(CrewAgentBase):
+    """Sub-agent mode / standalone single-agent Crew."""
+
+    FRAMEWORK = 'crewai'
+
+    def __init__(self, iGlobal: Any, *, process: Any = None, role: str = 'Assistant', task_description: str = '', goal: str = '', backstory: str = '', expected_output: str = ''):
```

Ruff (0.15.9)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)